### PR TITLE
ETFE-4516 Build TEMP04: delete-template

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -794,6 +798,8 @@ viewAllTemplates.noTemplates.p2 = You can create a template by submitting a draf
 viewAllTemplates.createANewMovement = Create a new movement
 viewAllTemplates.renamedNotification.title = Success
 viewAllTemplates.renamedNotification.content = Draft template "{0}" renamed to "{1}".
+viewAllTemplates.deletedNotification.title = Success
+viewAllTemplates.deletedNotification.content = Template ''{0}'' deleted.
```
Comment (stephenkward) [inline conf/messages]: Screenshot in jira shows this message with single quotes and with word Draft at the beginning.

```suggestion
viewAllTemplates.deletedNotification.content = Draft template '{0}' deleted.
```

File: test/controllers/draftTemplates/DeleteTemplateControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,129 @@
+package controllers.draftTemplates
```
Comment (stephenkward) [inline test/controllers/draftTemplates/DeleteTemplateControllerSpec.scala:17]: this file has no header info

File: it/connectors/DeleteDraftTemplateConnectorISpec.scala [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package connectors
```
Comment (stephenkward) [inline it/connectors/DeleteDraftTemplateConnectorISpec.scala:1]: this file has no header info

File: app/views/ViewAllTemplatesView.scala.html [hunk 4]
```diff
@@ -64,6 +63,20 @@
...
         }
 
+        @request.flash.get(TFE_DELETED_DRAFT_TEMPLATE_ID).map { templateName =>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    @notificationBanner(
+                        NotificationBanner(
+                            title = messages("viewAllTemplates.deletedNotification.title"),
+                            content = messages("viewAllTemplates.deletedNotification.content", templateName),
+                            classes = "govuk-notification-banner--success"
```
Comment (stephenkward) [inline app/views/ViewAllTemplatesView.scala.html:73]: The success banner width is not displayed correctly when the last template has been deleted.  see screenshot
Screenshot 2024-09-17 at 08 21 42

**Review by stephenkward — CHANGES_REQUESTED** [review]

File: conf/messages (hunk 1 above)
Comment (grahampaulcook) [reply conf/messages]: Change the quotes but the copy-deck doesn't say to start with the word draft
> Template '[templateName]' deleted.

Comment (grahampaulcook) [reply conf/messages]: https://github.com/hmrc/emcs-tfe-frontend/pull/241/commits/8b72bcc5b34e5fdc1ef60ccb79bd70a276573166 for the fix to the quotes on the flash message

File: app/views/ViewAllTemplatesView.scala.html (hunk 4 above)
Comment (grahampaulcook) [reply app/views/ViewAllTemplatesView.scala.html:73]: The `ViewAllTemplatesView` only sets full width content if there are templates else it sets it to two thirds

**Comment by grahampaulcook** [comment]
test this please

**Review by stephenkward — DISMISSED** [review]

**Review by stephenkward — APPROVED** [review]

**Pull request merged**